zenoh_app/teleop_autoware.py

```diff
@@ -47,13 +48,9 @@ def callback_status(sample):
         ###### Publishers
         self.publisher_gate_mode = self.session.declare_publisher(self.topic_prefix + SET_GATE_MODE_KEY_EXPR)
         self.publisher_gear = self.session.declare_publisher(self.topic_prefix + SET_GEAR_KEY_EXPR)
-        self.publisher_turn = self.session.declare_publisher(self.topic_prefix + SET_TURN_KEY_EXPR)
         self.publisher_control = self.session.declare_publisher(self.topic_prefix + SET_CONTROL_KEY_EXPR)
-        self.publisher_pedal = self.session.declare_publisher(self.topic_prefix + SET_PEDAL_CONTROL_KEY_EXPR)
-
-        ### Service
-        ###### Publishers
-        self.publisher_engage = self.session.declare_publisher(self.topic_prefix + SET_ENGAGE_KEY_EXPR)
+        # self.publisher_turn = self.session.declare_publisher(self.topic_prefix + SET_TURN_KEY_EXPR)
```

Comment:
Is there any reason why we don't need these?

Reply:
In [this PR](https://github.com/autowarefoundation/autoware-documentation/commit/919354b9839ec8834df6d633fbfb38f65090f058#diff-41eb8fe4e42bed00f5281b66e259425455572cc4f873f85042a99cf2442bc9b4), `autoware_auto_vehicle_msgs/msg/Engage` is replaced with `autoware_adapi_v1_msgs/srv/ChangeOperationMode` by AWF. Additionally, turn and pedal topics are not utilized in FMS. Instead, tire angle, longitudinal speed, and acceleration are delivered by publishing to `/external/selected/control_cmd`.